### PR TITLE
T9 f2 56 cleaning up create group

### PR DESCRIPTION
**Pull request by Kylefairbairn**
Should be fully functional

### CODE REVIEW

**Comment by Romangino** [comment]
Some conflicts need to be resolved I think

**Comment by mitchneides** [comment]
Maybe try using the auth service to pull the current user rather than slicing

File: src/components/create/index.js [hunk 1]
```diff
@@ -1,120 +1,222 @@
...
+        let userId = ''
+
+        if(allUsernames.length !== 0) {
+            user = await usersService.findUserByUsername(allUsernames[0])
+            userId = user._id
+        }
+
+        if(allUsernames.length > 1 && allAdmins.length !== 0 && invalidEntry !== true){
+
+            const group = {
```
Comment (mitchneides) [inline src/components/create/index.js]: try using the auth service here

Comment (Kylefairbairn) [reply src/components/create/index.js]: It is ready for merge now !

**Comment by Kylefairbairn** [comment]
Please re-view

**Comment by mitchneides** [comment]
Nice work good updates!

**Comment by Romangino** [comment]
Looks good now!

**Pull request merged**